components/prism-sparql.js

```diff
@@ -0,0 +1,17 @@
+Prism.languages.sparql = Prism.languages.extend('turtle', {
+		'variable': {pattern: /(\?|\$)[\w]+/,
```

Comment:
Is the first group supposed to be a lookbehind group?

If not, please you: `/[?$]\w+/`

Reply:
True, I've updated it.